server/configs/webapps/tomcat_lk.service

```diff
@@ -0,0 +1,28 @@
+# Systemd unit file for tomcat_lk
+
+[Unit]
+Description=lk Apache Tomcat Application
```

Comment:
Not a blocker for merging, but why is `lk` in lower case and abbreviated when the rest is title caps?

Reply:
These scripts are mostly just copied from Jon's samples